debug/gdbserver.py

```diff
@@ -1015,6 +1015,32 @@ def test(self):
             if privilege in self.supported:
                 assertEqual(actual, privilege)
 
+class ComplianceTest(GdbTest):
+
+    def test(self):
+        output = self.gdb.command("monitor riscv test_compliance")
+        #TODO: Fix OpenOCD to remove the -rtos riscv requirement
+        assert(("ALL TESTS PASSED") in output or ("Please run with -rtos riscv to run compliance test.") in output)
```

Comment:
It'd be nicer to do something like:
```
if "Please run with -rtos riscv to run compliance test." in output:
   return
assertIn("ALL TESTS PASSED", output)
```
which results in a slightly nicer error message if the tests didn't pass.